Django-CoWork-Connect/coapp/tasks.py

```diff
@@ -0,0 +1,34 @@
+from celery import shared_task
+from django.core.mail import send_mail
+from django.conf import settings
+from django.template.loader import render_to_string
+from django.utils.html import strip_tags
+from .models import Reservation
+
+@shared_task
+def send_confirmation_email(reservation_id, user_email):
+    reservation = Reservation.objects.get(id=reservation_id)
```

Comment:
a co jak rezerwacaj o zadanym id juz nie istnije . Bo np została odwołana, tutaj będzie nie złapany wyjatek. Uzyj filter i to obsłuz albo dodaj blo ktry